src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -695,6 +695,7 @@ def load_arguments(self, _):
                                                       help="The target location where the Bicep module will be published."))
         c.argument('documentationUri', arg_type=CLIArgumentType(options_list=['--documentationUri', '-d'],
                                                                 help="The documentation uri of the Bicep module."))
+        c.argument('force', arg_type=bicep_force_type, help="Allows overwriting an existing Bicep module version.")
```

Comment:
Please use the first-person voice in the help message
```suggestion
        c.argument('force', arg_type=bicep_force_type, help="Allow overwriting an existing Bicep module version.")
```

Reply:
Done.
Should I change the wording for  --force on decompile and restore as well?

Reply:
Could you give a specific example of what you want to change?

Reply:
There are three commands that has a --force parameter (`az bicep publish`, `az bicep decompile` and `az bicep restore`). The help message for the force parameters on decompile and restore uses the word 'Allows'